src/CommonLib/Processors/LDAPPropertyProcessor.cs

```diff
@@ -589,6 +489,167 @@ public async Task<ComputerProperties> ReadComputerProperties(ISearchResultEntry
 
             return props;
         }
+        
+        public static Dictionary<string, object> GetProperties(string ldapProperty, ISearchResultEntry entry)
+        {
```

Comment:
The goal here is single-truth LDAP property mapping.  It provides a dictionary of sorts of our available properties (the "how", see above) and tries to enforce that properties get added to `props` through our "vetted" source.